packages/cli/test/fixtures/discover/index.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+exports.SANDBOX_FILES = [
+  {
+    path: '',
+    file: 'test-schema-defs.json',
+    content: fs.readFileSync(
+      require.resolve('./models/test-schema-defs.json.txt'),
+    ),
+  },
+  {
+    path: 'dist/src/datasources',
+    file: 'mem.datasource.json',
+    content: fs.readFileSync(
+      require.resolve('./models/mem.datasource.json.txt'),
+    ),
+  },
+  {
+    path: 'dist/src/datasources',
+    file: 'mem.datasource.js',
+    content: fs.readFileSync(require.resolve('./models/mem.datasource.js.txt')),
```

Comment:
And similarly here, can we use `mem.datasource.js` please?

Reply:
Changing this extension to `.js` causes something to try and load the file, which then gets a `Cannot find module '@loopback/core'` error when I run `npm test`.  

The reason I had them all with the `.txt` extension is because they did it in `fixtures/repositories/models/`, is avoiding this error why?

Reply:
Fair enough, let's follow the current convention of using `.txt` files for now. Dealing with issues caused by using a different extension is out of scope of this pull request, I think.